infra/kops/TEMPLATE_CLUSTER/cluster.yml

```diff
@@ -3,36 +3,60 @@ kind: Cluster
 metadata:
   name: {{ .cluster_name }}
 spec:
+  additionalPolicies:
+    node: |
+      [
+        {
+          "Action": [
+            "sts:AssumeRole"
+          ],
+          "Effect": "Allow",
+          "Resource": "*"
+        },
+        {
+          "Effect": "Allow",
+          "Action": [
+            "ecr:DescribeImages",
+            "ecr:InitiateLayerUpload",
+            "ecr:UploadLayerPart",
+            "ecr:CompleteLayerUpload",
+            "ecr:PutImage",
+            "ecr:CreateRepository"
+          ],
+          "Resource": "*"
+        }
+      ]
   api:
     loadBalancer:
       type: Public
+  authorization:
+    alwaysAllow: {}
```

Comment:
Is this to disable authorisation on the k8s api?

Reply:
Yes, but not quite in the way the name suggests. This allows all authenticated API requests to perform any action, but it crucially still requires authentication - this is also the default authz method for Kubernetes <1.6, and the method we've been using so far. At some point we should look at using RBAC instead, but at the moment I'm trying to minimise the number of changes we make in one go.

More k8s authorization info here: https://kubernetes.io/docs/admin/authorization/

Reply:
So it authorise all requests as long as you're authenticated. It seems sensible to me to avoid too many changes at once 👍